counterpartylib/lib/config.py

```diff
@@ -88,7 +88,7 @@
 
 
 DEFAULT_REQUESTS_TIMEOUT = 20  # 20 seconds
-
+DEFAULT_RPC_BATCH_SIZE = 300
```

Comment:
yeah make sense! I increased the default value to 1000. I don't think it's necessary to tell to everyone to set `rpcthreads` to 1000: the default value is 3, and even if the batch query takes time, there is two other threads.